app/src/Pages/Portal/Expenses/index.js

```diff
@@ -19,9 +53,10 @@ class ExpensesPage extends React.Component {
 }
 export default connect(
   state => ({
+    currentUserId: getCurrentUserId(state),
     isListLoading: state.expenditures.isLoading,
     expendituresList: getExpendituresList(state),
     getGovOrCampIdAttributes: getGovOrCampIdAttributes(state),
   }),
   dispatch => ({ getExpenditures: data => dispatch(getExpenditures(data)) })
-)(ExpensesPage);
+)(withRouter(ExpensesPage));
```

Comment:
Why is `withRouter` necessary for this? I'm not questioning its validity here, just wanting to understand.

Reply:
Because that gives us access to the query parameters in url (`?status=draft`), so we can know what to load on first load